lib/mutant/reporter/hash/printer/config.rb

```diff
@@ -0,0 +1,36 @@
+module Mutant
+  class Reporter
+    class Hash
+      class Printer
+        # Printer for mutation config
+        class Config < self
+
+          # Report configuration
+          #
+          # @param [Mutant::Config] config
+          #
+          # @return [undefined]
+          #
+          # @api private
+          def run
+            matcher = object.matcher.inspect
+            # #<Mutant::Matcher::Config match_expressions: [Rumble::Loop]>
+            # Parse out expression [...]
+            expression = matcher.scan(/\[([\w:,\*]+)\]/).flatten.first
+            expression ||= matcher
```

Comment:
This will not work for complex matchers that are made off many expressions, as happens on projects without a namespace.

I suggest to serialize the full `Matcher::Config` object at the granularity it has instead of the first match expression.